e2e/vertical-full-page-map.spec.ts

```diff
@@ -3,54 +3,41 @@ import { test, expect } from '@playwright/test';
 test.describe('full page map test suite', () => {
   test.beforeEach(async ({page}) => {
     await page.goto('http://localhost:5042/locations_full_page_map');
-    await page.getByPlaceholder('Search for locations').click();
-  });
-
-  test('can search and get results', async ({ page }) => {
-    await page.getByPlaceholder('Search for locations').fill('virginia');
-    await page.getByPlaceholder('Search for locations').press('Enter');
-    const count = await page.locator('#js-answersVerticalResults').count();
-    expect(count).toBeGreaterThan(0);
   });
 
   test('clicking on a pin focuses on a result card', async ({ page }) => {
     await page.getByPlaceholder('Search for locations').fill('virginia');
     await page.getByPlaceholder('Search for locations').press('Enter');
     await page.getByRole('button', { name: 'Result number 5' }).click();
-    const results = page.locator('#js-answersVerticalResults div');
-    const count = await results.count();
-    let hasPinFocused = false;
-
-    for (var i = 0; i < count; i++) {
-      const currResult = await results.nth(i);
-      const classes = await currResult.evaluate(node => Array.from(node.classList));
-
-      if (classes.includes('yxt-Card--pinFocused')) {
-        hasPinFocused = true;
-        break;
-      }
-    }
-
-    expect(hasPinFocused).toBe(true);
+    await expect(page.locator('.yxt-Card--pinFocused')).toBeAttached();
```

Comment:
can we also check that this locator doesn't match anything before clicking the pin?